relude.cabal

```
@@ -183,7 +183,7 @@ test-suite relude-test
   main-is:             Spec.hs
 
   other-modules:       Test.Relude.Property
-
+                     , Test.Relude.Extra.Validation.Property
```

Commas are redundant in module definitions
```suggestion
                       Test.Relude.Extra.Validation.Property
```